<!DOCTYPE html>
<html lang="en-us">
    
    


    <head>
    <link href="https://gmpg.org/xfn/11" rel="profile">
    <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="Cache-Control" content="public" />
<!-- Enable responsiveness on mobile devices -->
<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">
<meta name="generator" content="Hugo 0.57.2" />

    
    
    

<title> â€¢ Functional Analysis</title>


<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content=""/>
<meta name="twitter:description" content="Activities - TO MERGE INTO WORKFLOW Activity - Getting a list of DE genes SH-SY5Y is a widely-used neuroblastoma cell line. With approriate treatment, they can be readily differentiated from their base state into a more neuronal phenotype.
In their paper Transcriptomic Profiling Discloses Molecular and Cellular Events Related to Neuronal Differentiation in SH-SY5Y Neuroblastoma Cells Pezzini et al induced differentiation of the SH-SY5Y Neuroblastoma cell line and measured transcriptomic changes via RNAseq (Pezzini et al."/>

<meta property="og:title" content="" />
<meta property="og:description" content="Activities - TO MERGE INTO WORKFLOW Activity - Getting a list of DE genes SH-SY5Y is a widely-used neuroblastoma cell line. With approriate treatment, they can be readily differentiated from their base state into a more neuronal phenotype.
In their paper Transcriptomic Profiling Discloses Molecular and Cellular Events Related to Neuronal Differentiation in SH-SY5Y Neuroblastoma Cells Pezzini et al induced differentiation of the SH-SY5Y Neuroblastoma cell line and measured transcriptomic changes via RNAseq (Pezzini et al." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://monashbioinformaticsplatform.github.io/enrichment_analysis_workshop/module/activities/" />
<meta property="og:site_name" content="Enrichment Analysis" />


    


<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">








<link rel="stylesheet" href="/enrichment_analysis_workshop/scss/hyde-hyde.6a83d62c39a364f036df4db1ecd564645635d6c7fc182425cb501218fec485f5.css" integrity="sha256-aoPWLDmjZPA2302x7NVkZFY11sf8GCQly1ASGP7EhfU=">


<link rel="stylesheet" href="/enrichment_analysis_workshop/scss/print.2744dcbf8a0b2e74f8a50e4b34e5f441be7cf93cc7de27029121c6a09f9e77bc.css" integrity="sha256-J0Tcv4oLLnT4pQ5LNOX0Qb58&#43;TzH3icCkSHGoJ&#43;ed7w=" media="print">




<link rel="stylesheet" href="/enrichment_analysis_workshop/scss/hugo-toc.8549a64d301421b4256ea9a0d5ca8b99178799f569d5d2daeb95f24cef3ce6b7.css" integrity="sha256-hUmmTTAUIbQlbqmg1cqLmReHmfVp1dLa65XyTO885rc=">



    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
    <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
    <!-- Icons -->
    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/apple-touch-icon-144-precomposed.png">
    <link rel="shortcut icon" href="/favicon.png">
    
</head>


    <body class=" ">
    
<div class="sidebar">
  <div class="container ">
    <div class="sidebar-about">
      <span class="site__title">
        <a href="https://monashbioinformaticsplatform.github.io/enrichment_analysis_workshop/">Functional Analysis</a>
      </span>
      
        
        
        
        <div class="author-image">
          <img src="https://monashbioinformaticsplatform.github.io/enrichment_analysis_workshop/img/mbp_logo.png" alt="Author Image" class="img--circle img--headshot element--center">
        </div>
        
      
      
      <p class="site__description">
        
      </p>
    </div>
    <div class="collapsible-menu">
      <input type="checkbox" id="menuToggle">
      <label for="menuToggle">Functional Analysis</label>
      <div class="menu-content">
        <div>
	<ul class="sidebar-nav">
		 
		 
			 
				<li>
					<a href="module/overview">
						<span>01 Overview</span>
					</a>
				</li>
			 
		 
			 
				<li>
					<a href="/enrichment_analysis_workshop/module/basics">
						<span>02 Basics</span>
					</a>
				</li>
			 
		 
			 
				<li>
					<a href="/enrichment_analysis_workshop/module/example-data-preparation">
						<span>03 Supplementary: Example Data</span>
					</a>
				</li>
			 
		
	</ul>
</div>

        <section class="social">
	
	<a href="https://twitter.com/MonashBioinfo" rel="me"><i class="fab fa-twitter fa-lg" aria-hidden="true"></i></a>
	
	
	
	<a href="https://github.com/MonashBioinformaticsPlatform" rel="me"><i class="fab fa-github fa-lg" aria-hidden="true"></i></a>
	
	
	
	
	
	
	
	
	
	
	
	
	
	
</section>

      </div>
    </div>
    


  </div>
</div>

        <div class="content container">
            
    
<article>
  <header>
    <h1></h1>
    
    
<div class="post__meta">
    
    
    
    
    
    
    <br/>
    <i class="fas fa-clock"></i> 4 min read
</div>


  </header>
  
  
  
    <div class="toc-wrapper">
      <input type="checkbox" id="tocToggle">
      <label for="tocToggle">Table of Content</label>
      
          <nav id="TableOfContents">
<ul>
<li><a href="#activities-to-merge-into-workflow">Activities  - TO MERGE INTO WORKFLOW</a>
<ul>
<li><a href="#activity-getting-a-list-of-de-genes">Activity - Getting a list of DE genes</a></li>
<li><a href="#enrichment-stats-interactively">Enrichment stats interactively</a></li>
<li><a href="#enrichment-in-gprofiler">Enrichment in gProfiler?</a></li>
<li><a href="#enrichment-in-david">Enrichment in DAVID</a></li>
<li><a href="#explore-in-string">Explore in String?</a></li>
<li><a href="#explore-in-reactome">Explore in Reactome?</a></li>
<li><a href="#end-of-session-challenge-dr-dodge-s-experiment">End of session Challenge : Dr Dodge&rsquo;s Experiment</a></li>
</ul></li>
</ul>
</nav>
      
    </div>
  
  
  <div class="post">
    

<h1 id="activities-to-merge-into-workflow">Activities  - TO MERGE INTO WORKFLOW</h1>

<h2 id="activity-getting-a-list-of-de-genes">Activity - Getting a list of DE genes</h2>

<p>SH-SY5Y is a widely-used neuroblastoma cell line.
With approriate treatment, they can be readily differentiated from their base state
into a more neuronal phenotype.</p>

<p>In their paper <a href="https://link.springer.com/article/10.1007%2Fs10571-016-0403-y"><em>Transcriptomic Profiling Discloses Molecular and Cellular Events Related to Neuronal Differentiation in SH-SY5Y Neuroblastoma Cells</em></a>
Pezzini et al induced differentiation of the SH-SY5Y Neuroblastoma cell line and measured transcriptomic changes via RNAseq (Pezzini et al. 2017).</p>

<p>Here are some differential expression results from their raw data:
<a href="http://degust.erc.monash.edu/degust/compare.html?code=5b2c7805ab8f8c5f2dc8c72e61b049b0#?plot=mds">http://degust.erc.monash.edu/degust/compare.html?code=5b2c7805ab8f8c5f2dc8c72e61b049b0#?plot=mds</a></p>

<ol>
<li>Download the full table of data from degust.  Import into excel.</li>
</ol>

<!-- File>Import -->

<ol>
<li><p>QUESTION: How many genes are differentially expressed?</p>

<ul>
<li>Significant at 0.01? (FDR Column = Corrected P-value)</li>

<li><p>And because there are so many in this experiment (its a massive change to cells) -  At 2-fold up/down regulated?
NB: The &lsquo;differentiated&rsquo; column shows the log2 fold-change of differentiated cells vs untreated cells (log2(diff)-log2(undiff)). 0 is unchanged.</p></li>

<li><p>And (just for this exercise!) -  take significance at p&lt;1x10^-7  , with 2x fold change? This is a ridiculous threhsold - typically you would only filtere at p&lt;0.01 (and occasionally 2fold change) but the difference between the two cell conditions is pretty extreme! This threshold gives a more normal number of de genes for downstream analysis.</p></li>
</ul></li>
</ol>

<p>ANSWER:
    - 6391 ( this is massive )
    - 2412 (still very large!)
    - 168  (a more typical result, by using a stupid threshold)</p>

<ol>
<li>QUESTION: How many genes are <em>tested</em>? This is your background.

<ul>
<li>But with ~20k human genes - why are there genes missing?
<br /></li>
</ul></li>
</ol>

<p>ANSWER: 14420</p>

<ol>
<li>COMMON GOTCHA: Can you find SEPT4?

<ul>
<li>Because <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1044-7"><em>Gene name errors are widespread in the scientific literature</em></a></li>
<li>You can&rsquo;t fix it automatically (try converting it to text!). You have to avoid it by importing gene columns as &lsquo;text&rsquo; columns in excel. See video from HUGO : <a href="https://www.genenames.org/help/faq/#!/#tocAnchor-1-25-1">https://www.genenames.org/help/faq/#!/#tocAnchor-1-25-1</a></li>
<li>NB: You can ignore these for this workshop, but you want this to be right for publication!</li>
</ul></li>
</ol>

<h2 id="enrichment-stats-interactively">Enrichment stats interactively</h2>

<p>Check out the toy enrichment calculater <a href="https://bioinformatics3.erc.monash.edu/rsconnect/connect/#/apps/40/access">here</a></p>

<ol>
<li><p>QUESTION: If 17 of the 168 differentially expressed genes are annotated with the 500-gene term of interest. Is it significant?</p>

<ul>
<li>Smaller backround (e.g. proteomic dataset)</li>
<li>Fewer annotation terms tested?</li>
</ul></li>

<li><p>QUESTION : How many of the of the 168 differentially expressed genes have to be annotated with a 500-gene term of interest for it to be significant at p=0.05?</p>

<ul>
<li>What about a term with 30 annotated genes?</li>
</ul></li>
</ol>

<p>ANSWER
    - No, corrected pval=0.217
    - 19 genes == 11% bs the 3.5% of bg
    - 5 / 30 genes = ~3% vs 0.2% of bg</p>

<p>NB: This is just a toy calculator for this training, it is quite constrained. Bonus points if you break it. There are likely many better ones online e.g. : <a href="https://keisan.casio.com/exec/system/1180573201">https://keisan.casio.com/exec/system/1180573201</a></p>

<h2 id="enrichment-in-gprofiler">Enrichment in gProfiler?</h2>

<p>Try gprofiler GOSt for functional enrichment: <a href="https://biit.cs.ut.ee/gprofiler/gost">https://biit.cs.ut.ee/gprofiler/gost</a></p>

<p>Using the 168 differentially expressed gene list.</p>

<p>NB: Set the background gene list with _Advanced Options &gt; Statistical Domain scope &gt; Custom : Paste in genelist _</p>

<h2 id="enrichment-in-david">Enrichment in DAVID</h2>

<p><em>(Maybe skip?) Main point is that different tools give different results despite using the same databases! Though david has some nice features.</em></p>

<p>Try calculating funtional enrichment in DAVID:  <a href="https://david.ncifcrf.gov/">https://david.ncifcrf.gov/</a></p>

<p>Use the 168 differentially expressed gene list and the background set.</p>

<p>NB: Need to use ensembl Ids (e.g. ENSG00000170075) for a background in DAVID, it won&rsquo;t allow gene names due to potential ambiguity. Select &ldquo;ENSEMBL_GENE_ID&rdquo; for id type.</p>

<ol>
<li><p>QUESTION : Look at the results for the GO terms. Why is it different to gProfiler?</p></li>

<li><p>QUESTION : Look at the KEGG results for Cellular adhesion molecules. Its not significant, but is it useful?</p></li>
</ol>

<p>ANSWER:
  - Same DB, different tools. GO is a tree and gets carved in differen ways.
  - P-value isn&rsquo;t everything. Those few genes are hitting interactiving molecules within the neural system. The immune system is irrevant to this experiment.</p>

<h2 id="explore-in-string">Explore in String?</h2>

<p>Exlore protein-protein interactions with STRING: <a href="https://string-db.org">https://string-db.org</a></p>

<p>Use the 168 differentially expressed genes. No background needed for this one, not interested in calculating the enrichment.</p>

<p>QUESTION: Click through on the interactions to see what the evidence for a protein-protein interaction is. Do you trust that interaction?</p>

<h2 id="explore-in-reactome">Explore in Reactome?</h2>

<p>Expore enrichment in the the pathway browser of reactome : <a href="https://reactome.org/">https://reactome.org/</a></p>

<p>Go to &lsquo;analyse data&rsquo; and paste the 168 differentially expressed genes. No background.</p>

<p>QUESTION: Click through &lsquo;Syndecan Interactions&rsquo; And expand it one the left hand tree.</p>

<p>QUESTION: Go back to <a href="https://reactome.org/">https://reactome.org/</a> and search for your favourite pathway/process/gene. Is it covered?</p>

<h2 id="end-of-session-challenge-dr-dodge-s-experiment">End of session Challenge : Dr Dodge&rsquo;s Experiment</h2>

<p>There&rsquo;s quite a few ways you can calculate functional enrichment - so there&rsquo;s always a risk of <a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002106">p-hacking</a>, wheather accidental or deliberate.</p>

<p><em>So lets go deliberate&hellip;</em></p>

<p>Dr Dodge is convinced that your experiment should turn out with relevants to GO term &lsquo;segmentation&rsquo; (don&rsquo;t ask why..) but p=0.06! But they must be significant, the paper has to be submitted by Friday!</p>

<p>What hacky and statsticically questionable approaches could you apply to call these results &ldquo;significant&rdquo;?</p>

  </div>
  

<div class="navigation navigation-single">
    
    
    <a href="/enrichment_analysis_workshop/module/example-data-preparation/" class="navigation-next">
      <span class="navigation-tittle">Example Data Preparation</span>
      <i aria-hidden="true" class="fa fa-chevron-right"></i>
    </a>
    
</div>


  

  
    


</article>


        </div>
        
    

<script defer src="https://use.fontawesome.com/releases/v5.5.0/js/all.js" integrity="sha384-GqVMZRt5Gn7tB9D9q7ONtcp4gtHIUEW/yG7h98J7IpE3kpi+srfFyyB/04OV6pG0" crossorigin="anonymous"></script>


    
    
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.13.1/highlight.min.js"></script>
        
    
    <script type="text/javascript">
        
        hljs.initHighlightingOnLoad();
    </script>
    






    



    </body>
</html>
