<!DOCTYPE html>
<html lang="en" class="js csstransforms3d">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="generator" content="Hugo 0.123.8">
    <meta name="description" content="">


    <link rel="icon" href="/enrichment_analysis_workshop/images/favicon.png" type="image/png">

    <title>Example Data Preparation :: Functional Analysis</title>

    
    <link href="/enrichment_analysis_workshop/css/nucleus.css?1710473129" rel="stylesheet">
    <link href="/enrichment_analysis_workshop/css/fontawesome-all.min.css?1710473129" rel="stylesheet">
    <link href="/enrichment_analysis_workshop/css/hybrid.css?1710473129" rel="stylesheet">
    <link href="/enrichment_analysis_workshop/css/featherlight.min.css?1710473129" rel="stylesheet">
    <link href="/enrichment_analysis_workshop/css/perfect-scrollbar.min.css?1710473129" rel="stylesheet">
    <link href="/enrichment_analysis_workshop/css/auto-complete.css?1710473129" rel="stylesheet">
    <link href="/enrichment_analysis_workshop/css/atom-one-dark-reasonable.css?1710473129" rel="stylesheet">
    <link href="/enrichment_analysis_workshop/css/theme.css?1710473129" rel="stylesheet">
    <link href="/enrichment_analysis_workshop/css/tabs.css?1710473129" rel="stylesheet">
    <link href="/enrichment_analysis_workshop/css/hugo-theme.css?1710473129" rel="stylesheet">
    
    <link href="/enrichment_analysis_workshop/css/theme-green.css?1710473129" rel="stylesheet">
    
    

    <script src="/enrichment_analysis_workshop/js/jquery-3.3.1.min.js?1710473129"></script>

    <style>
      :root #header + #content > #left > #rlblock_left{
          display:none !important;
      }
      
    </style>
    
  </head>
  <body class="" data-url="/enrichment_analysis_workshop/supplementary/example-data-preparation/">
    <nav id="sidebar" class="">



  <div id="header-wrapper">
    <div id="header">
      <a href="https://www.monash.edu/researchinfrastructure/bioinformatics" target="_blank"><img src="https://monashbioinformaticsplatform.github.io/enrichment_analysis_workshop/img/mbp_full_name.png" width="100%" style="margin-top: 15px;"></a>

    </div>
    
  </div>
  
    <section id="homelinks">
      <ul>
        <li>
            <a class="padding" href='/'><i class='fas fa-home'></i> Home</a>
        </li>
      </ul>
    </section>
  

    <div class="highlightable">
    <ul class="topics">

        
          
          




 
  
    
    <li data-nav-id="/enrichment_analysis_workshop/overview/" title="Overview" class="dd-item
        
        
        
        ">
      <a href="/enrichment_analysis_workshop/overview/">
          Overview
          
      </a>
      
      
    </li>
  
 

          
          




 
  
    
    <li data-nav-id="/enrichment_analysis_workshop/content/" title="Workshop material" class="dd-item
        
        
        
        ">
      <a href="/enrichment_analysis_workshop/content/">
          Workshop material
          
      </a>
      
      
        <ul>
          
          
          

        
          
            
            




 
  
    
      <li data-nav-id="/enrichment_analysis_workshop/content/stats/" title="Enrichment Statistics" class="dd-item ">
        <a href="/enrichment_analysis_workshop/content/stats/">
        Enrichment Statistics
        
        </a>
    </li>
     
  
 

            
          
            
            




 
  
    
      <li data-nav-id="/enrichment_analysis_workshop/content/example-dataset/" title="Example Analysis" class="dd-item ">
        <a href="/enrichment_analysis_workshop/content/example-dataset/">
        Example Analysis
        
        </a>
    </li>
     
  
 

            
          
            
            




 
  
    
      <li data-nav-id="/enrichment_analysis_workshop/content/genelists/" title="Defining the genelist" class="dd-item ">
        <a href="/enrichment_analysis_workshop/content/genelists/">
        Defining the genelist
        
        </a>
    </li>
     
  
 

            
          
            
            




 
  
    
      <li data-nav-id="/enrichment_analysis_workshop/content/enrichment/" title="Enrichment Tools" class="dd-item ">
        <a href="/enrichment_analysis_workshop/content/enrichment/">
        Enrichment Tools
        
        </a>
    </li>
     
  
 

            
          
            
            




 
  
    
      <li data-nav-id="/enrichment_analysis_workshop/content/reporting/" title="Reporting results" class="dd-item ">
        <a href="/enrichment_analysis_workshop/content/reporting/">
        Reporting results
        
        </a>
    </li>
     
  
 

            
          
            
            




 
  
    
      <li data-nav-id="/enrichment_analysis_workshop/content/resources/" title="Resources" class="dd-item ">
        <a href="/enrichment_analysis_workshop/content/resources/">
        Resources
        
        </a>
    </li>
     
  
 

            
          
            
            




 
  
    
      <li data-nav-id="/enrichment_analysis_workshop/content/challenge/" title="End of Session Challenge" class="dd-item ">
        <a href="/enrichment_analysis_workshop/content/challenge/">
        End of Session Challenge
        
        </a>
    </li>
     
  
 

            
          
            
            




 
  
    
      <li data-nav-id="/enrichment_analysis_workshop/content/end/" title="Feedback" class="dd-item ">
        <a href="/enrichment_analysis_workshop/content/end/">
        Feedback
        
        </a>
    </li>
     
  
 

            
          
        
        </ul>
      
    </li>
  
 

          
          




 
  
    
    <li data-nav-id="/enrichment_analysis_workshop/supplementary/" title="Supplementary" class="dd-item
        parent
        
        
        ">
      <a href="/enrichment_analysis_workshop/supplementary/">
          Supplementary
          
      </a>
      
      
        <ul>
          
          
          

        
          
            
            




 
  
    
      <li data-nav-id="/enrichment_analysis_workshop/supplementary/example-data-preparation/" title="Example Data Preparation" class="dd-item active">
        <a href="/enrichment_analysis_workshop/supplementary/example-data-preparation/">
        Example Data Preparation
        
        </a>
    </li>
     
  
 

            
          
        
        </ul>
      
    </li>
  
 

          
        
    </ul>

    
    

    
    <section id="footer">
      <p>Built with <a href="https://github.com/matcornic/hugo-theme-learn"><i class="fas fa-heart"></i></a> from <a href="https://getgrav.org">Grav</a> and <a href="https://gohugo.io/">Hugo</a></p>

    </section>
  </div>
</nav>




        <section id="body">
        <div id="overlay"></div>
        <div class="padding highlightable">
              
              <div>
                <div id="top-bar">
                
                
                <div id="breadcrumbs" itemscope="" itemtype="http://data-vocabulary.org/Breadcrumb">
                    <span id="sidebar-toggle-span">
                        <a href="#" id="sidebar-toggle" data-sidebar-toggle="">
                          <i class="fas fa-bars"></i>
                        </a>
                    </span>
                  
                  <span id="toc-menu"><i class="fas fa-list-alt"></i></span>
                  
                  <span class="links">
                 
                 
                    
          
          
            
            
          
          
            
            
          
          
            <a href='/enrichment_analysis_workshop/'>Functional Analysis of -Omics Data</a> > <a href='/enrichment_analysis_workshop/supplementary/'>Supplementary</a> > Example Data Preparation
          
        
          
        
          
        
                 
                  </span>
                </div>
                
                    <div class="progress">
    <div class="wrapper">
<nav id="TableOfContents">
  <ul>
    <li><a href="#sequence-data-download">Sequence data download</a></li>
    <li><a href="#reference-data-download">Reference data download</a></li>
    <li><a href="#align-and-count">Align and count</a></li>
    <li><a href="#differential-expression">Differential expression</a></li>
  </ul>
</nav>
    </div>
</div>

                
              </div>
            </div>
            
        <div id="head-tags">
        
        </div>
        
        <div id="body-inner">
          
            <h1>
              
              Example Data Preparation
            </h1>
          

        


<p>SH-SY5Y is a widely-used neuroblastoma cell line.
With approriate treatment, they can be readily differentiated from their base state
into a more neuronal phenotype.</p>
<p>In their paper <a href="https://link.springer.com/article/10.1007%2Fs10571-016-0403-y"><em>Transcriptomic Profiling Discloses Molecular and Cellular Events Related to Neuronal Differentiation in SH-SY5Y Neuroblastoma Cells</em></a>
Pezzini et al induced differentiation of the SH-SY5Y Neuroblastoma cell line and measured transcriptomic changes via RNAseq (Pezzini et al. 2017).</p>
<hr>
<p>This document describes a simple reanalysis of their raw data, to produce the
results browseable through degust <a href="http://degust.erc.monash.edu/degust/compare.html?code=5b2c7805ab8f8c5f2dc8c72e61b049b0#?plot=mds">here</a></p>
<h1 id="methods">Methods</h1>
<h2 id="sequence-data-download">Sequence data download</h2>
<p>Data is available in SRA with accession <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE77383">GSE77383</a>.</p>
<div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4;"><code class="language-sh" data-lang="sh"><span style="display:flex;"><span><span style="color:#75715e"># With SRA toolkit 2.8.0</span>
</span></span><span style="display:flex;"><span><span style="color:#75715e"># module load sratoolkit/2.8.0</span>
</span></span><span style="display:flex;"><span>mkdir fastq
</span></span><span style="display:flex;"><span>cd fastq
</span></span><span style="display:flex;"><span>fastq-dump -I --split-files -A SRR3133925_nodiff_rep1 --gzip SRR3133925
</span></span><span style="display:flex;"><span>fastq-dump -I --split-files -A SRR3133926_nodiff_rep2 --gzip SRR3133926
</span></span><span style="display:flex;"><span>fastq-dump -I --split-files -A SRR3133927_nodiff_rep3 --gzip SRR3133927
</span></span><span style="display:flex;"><span>fastq-dump -I --split-files -A SRR3133928_diff_rep1 --gzip SRR3133928
</span></span><span style="display:flex;"><span>fastq-dump -I --split-files -A SRR3133929_diff_rep2 --gzip SRR3133929
</span></span><span style="display:flex;"><span>fastq-dump -I --split-files -A SRR3133930_diff_rep3 --gzip SRR3133930
</span></span></code></pre></div><h2 id="reference-data-download">Reference data download</h2>
<p>Also downloaded reference genome and genes gtf from <a href="http://www.ensembl.org/info/data/ftp/index.html">ensembl</a>
(GRCh38, ensembl release 96).</p>
<div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4;"><code class="language-sh" data-lang="sh"><span style="display:flex;"><span>mkdir ref_data
</span></span><span style="display:flex;"><span>cd ref_data
</span></span><span style="display:flex;"><span>wget ftp://ftp.ensembl.org/pub/release-96/gtf/homo_sapiens/Homo_sapiens.GRCh38.96.gtf.gz
</span></span><span style="display:flex;"><span>wget ftp://ftp.ensembl.org/pub/release-96/fasta/homo_sapiens/dna/Homo_sapiens.GRCh38.dna_sm.primary_assembly.fa.gz
</span></span><span style="display:flex;"><span>gunzip Homo_sapiens.GRCh38.96.gtf.gz
</span></span></code></pre></div><h2 id="align-and-count">Align and count</h2>
<p>Using <a href="https://monashbioinformaticsplatform.github.io/RNAsik-pipe/">RNAsik</a> (Tsyganov et al. 2018)
pipeline to run QC, alignment and gene counting on these fastqs.</p>
<p>This runs <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">fastQC</a>,
<a href="https://multiqc.info/">multiQC</a> (Ewels et al. 2016), <a href="https://broadinstitute.github.io/picard/">picard</a>, STAR alignment (Dobin et al. 2013) and featureCounts (Liao et al. 2014).</p>
<div class="highlight"><pre tabindex="0" style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4;"><code class="language-sh" data-lang="sh"><span style="display:flex;"><span><span style="color:#75715e"># Using RNAsik version 1.5.0</span>
</span></span><span style="display:flex;"><span>RNAsik -fqDir fastqs -counts -paired -qc -multiqc -align star -fastaRef ref_data/Homo_sapiens.GRCh38.dna_sm.primary_assembly.fa.gz -gtfFile ref_data/Homo_sapiens.GRCh38.96.gtf -pairIds _1,_2
</span></span></code></pre></div><h2 id="differential-expression">Differential expression</h2>
<p>The counts table was uploaded to <a href="http://degust.erc.monash.edu/">Degust</a> for
processing.
Genes with at least 10 reads in at least one sample were included,
and limma/voom used for differential expression
analysis (Ritchie et al. 2015, Law et al. 2014).</p>
<p>Dataset results acccessible :  <a href="http://degust.erc.monash.edu/degust/compare.html?code=5b2c7805ab8f8c5f2dc8c72e61b049b0#?plot=mds">http://degust.erc.monash.edu/degust/compare.html?code=5b2c7805ab8f8c5f2dc8c72e61b049b0#?plot=mds</a></p>
<h1 id="references">References</h1>
<p>Dobin, Alexander, Carrie A. Davis, Felix Schlesinger, Jorg Drenkow, Chris Zaleski, Sonali Jha, Philippe Batut, Mark Chaisson, and Thomas R. Gingeras. 2013. “STAR: Ultrafast universal RNA-seq aligner.” Bioinformatics 29 (1): 15–21. <a href="https://doi.org/10.1093/bioinformatics/bts635">https://doi.org/10.1093/bioinformatics/bts635</a>.</p>
<p>Ewels, Philip, Måns Magnusson, Sverker Lundin, and Max Käller. 2016. “MultiQC: Summarize analysis results for multiple tools and samples in a single report.” Bioinformatics 32 (19): 3047–8. <a href="https://doi.org/10.1093/bioinformatics/btw354">https://doi.org/10.1093/bioinformatics/btw354</a>.</p>
<p>Law, Charity W, Yunshun Chen, Wei Shi, and Gordon K Smyth. 2014. “voom: Precision weights unlock linear model analysis tools for RNA-seq read counts.” Genome Biology 15 (2): R29. <a href="https://doi.org/10.1186/gb-2014-15-2-r29">https://doi.org/10.1186/gb-2014-15-2-r29</a>.</p>
<p>Liao, Yang, Gordon K. Smyth, and Wei Shi. 2014. “FeatureCounts: An efficient general purpose program for assigning sequence reads to genomic features.” Bioinformatics 30 (7): 923–30. <a href="https://doi.org/10.1093/bioinformatics/btt656">https://doi.org/10.1093/bioinformatics/btt656</a>.</p>
<p>Pezzini, Francesco, Laura Bettinetti, Francesca Di Leva, Marzia Bianchi, Elisa Zoratti, Rosalba Carrozzo, Filippo M. Santorelli, Massimo Delledonne, Maciej Lalowski, and Alessandro Simonati. 2017. “Transcriptomic Profiling Discloses Molecular and Cellular Events Related to Neuronal Differentiation in SH-SY5Y Neuroblastoma Cells.” Cellular and Molecular Neurobiology 37 (4). Springer US: 665–82. <a href="https://doi.org/10.1007/s10571-016-0403-y">https://doi.org/10.1007/s10571-016-0403-y</a>.</p>
<p>Ritchie, Matthew E., Belinda Phipson, Di Wu, Yifang Hu, Charity W. Law, Wei Shi, and Gordon K. Smyth. 2015. “limma powers differential expression analyses for RNA-sequencing and microarray studies.” Nucleic Acids Research 43 (7): e47. <a href="https://doi.org/10.1093/nar/gkv007">https://doi.org/10.1093/nar/gkv007</a>.</p>
<p>Tsyganov, Kirill, Andrew James Perry, Stuart Kenneth Archer, and David Powell. 2018. “RNAsik: A Pipeline for complete and reproducible RNA-seq analysis that runs anywhere with speed and ease.” Journal of Open Source Software 3 (28): 583. <a href="https://doi.org/10.21105/joss.00583">https://doi.org/10.21105/joss.00583</a>.</p>


<footer class="footline">
	
</footer>

        
        </div>
        

      </div>

    <div id="navigation">
        
        

        
            
            
                
                    
                    
                
                

                    
                    
                        
                    
                    

                    
                        
            
            
                
                    
                        
                        
                    
                
                

                    
                    
                    

                    
            
        
                    
                        
            
            
                
                    
                
                

                    
                    
                    

                    
                        
            
            
                
                    
                
                

                    
                    
                    

                    
            
        
                    
                        
            
            
                
                    
                
                

                    
                    
                    

                    
            
        
                    
                        
            
            
                
                    
                
                

                    
                    
                    

                    
            
        
                    
                        
            
            
                
                    
                
                

                    
                    
                    

                    
            
        
                    
                        
            
            
                
                    
                
                

                    
                    
                    

                    
            
        
                    
                        
            
            
                
                    
                
                

                    
                    
                    

                    
            
        
                    
                        
            
            
                
                    
                
                

                    
                    
                    

                    
            
        
                    
                        
            
            
                
                    
                
                

                    
                    
                    

                    
            
        
                    
            
        
                    
                        
            
            
                
                    
                    
                
                

                    
                    
                    

                    
                        
            
            
                
                    
                    
                
                

                    
                    
                    

                    
            
        
                    
            
        
                    
            
        

        


	 
	 
		
			<a class="nav nav-prev" href="/enrichment_analysis_workshop/supplementary/" title="Supplementary"> <i class="fa fa-chevron-left"></i></a>
		
		
			<a class="nav nav-next" href="/enrichment_analysis_workshop/overview/" title="Overview" style="margin-right: 0px;"><i class="fa fa-chevron-right"></i></a>
		
	
    </div>

    </section>

    <div style="left: -1000px; overflow: scroll; position: absolute; top: -1000px; border: none; box-sizing: content-box; height: 200px; margin: 0px; padding: 0px; width: 200px;">
      <div style="border: none; box-sizing: content-box; height: 200px; margin: 0px; padding: 0px; width: 200px;"></div>
    </div>
    <script src="/enrichment_analysis_workshop/js/clipboard.min.js?1710473129"></script>
    <script src="/enrichment_analysis_workshop/js/perfect-scrollbar.min.js?1710473129"></script>
    <script src="/enrichment_analysis_workshop/js/perfect-scrollbar.jquery.min.js?1710473129"></script>
    <script src="/enrichment_analysis_workshop/js/jquery.sticky.js?1710473129"></script>
    <script src="/enrichment_analysis_workshop/js/featherlight.min.js?1710473129"></script>
    <script src="/enrichment_analysis_workshop/js/highlight.pack.js?1710473129"></script>
    <script>hljs.initHighlightingOnLoad();</script>
    <script src="/enrichment_analysis_workshop/js/modernizr.custom-3.6.0.js?1710473129"></script>
    <script src="/enrichment_analysis_workshop/js/learn.js?1710473129"></script>
    <script src="/enrichment_analysis_workshop/js/hugo-learn.js?1710473129"></script>
    
        
            <script src="/enrichment_analysis_workshop/mermaid/mermaid.js?1710473129"></script>
        
        <script>
            mermaid.initialize({ startOnLoad: true });
        </script>
    
    

  </body>
</html>

